components/src/Guidelines/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from 'react'
+import { Styled, Text as ThemeUIText } from 'theme-ui'
+import { Box } from 'theme-ui'
+
+export const Guidelines = ({ children }: ListProp) => (
+    <Box>
+        {children}
+    </Box>
+)
+
+const Dos = ({ children, title, icon }: ListProps) => (
+    <ThemeUIText
+        sx={{
+            fontFamily: 'sans serif',
+            lineHeight: 2,
+            letterSpacing: 1,
+
+        }}>
+
+        <ThemeUIText as="h3">{icon}{title}</ThemeUIText>
+        <Styled.ul>
+            {children}
+        </Styled.ul>
+
+    </ThemeUIText>
+)
+
+const Donts = ({ children, title, icon }: ListProps) => (
+    <ThemeUIText
+        sx={{
+            fontFamily: 'sans serif',
+            lineHeight: 2,
+            letterSpacing: 1,
+        }}>
+
+        <ThemeUIText as="h3">{icon}{title}</ThemeUIText>
+        <Styled.ul>
+            {children}
+        </Styled.ul>
+
+    </ThemeUIText>
+)
```

Comment:
Can you see that these components are largely the same? They're actually the same code for now, but they'll have different defaults as defined in #8.

Here's a suggestion: create a component named `GuidelinesList` that contains this code and then you can define `Dos` and `Donts` with their respective defaults. It should look something like this:
```jsx
const Dos = ({ title = 'Dos', ... }) => <GuidelinesList title={title} ... />
```